contentcuration/contentcuration/static/js/edit_channel/import/views/filters.js

```diff
@@ -0,0 +1,16 @@
+function pluralize(number, singular, plural) {
```

Comment:
Might be worth consolidating functions with the "format_count" function under handlebars/helpers.js to avoid duplicate code (not needed for now though)